scribble/app/controllers/posts_controller.rb

```diff
@@ -0,0 +1,40 @@
+class PostsController < ApplicationController
+  def index
+    @posts = Post.all
+  end
+
+  def new
+    @post = Post.new
+  end
+
+  def edit
+    @post = Post.find(params[:id])
+  end
+
+  def show
+    @post = Post.find(params[:id])
+    @comments = @post.comments
+  end
+
+  def create
+    @post = Post.create!(posts_params)
+    redirect_to @post
+  end
+
+  def update
+    @post = Post.find(params[:id])
+    @post.update(posts_params)
+    redirect_to @post
+  end
+
+  def destroy
+    @post = Post.find(params[:id])
```

Comment:
Could you write a `before_action` so you don't need to write `@post = Post.find(params[:id])` a zillion times?